.github/workflows/ci.yml

```
@@ -20,6 +20,7 @@ jobs:
         ruby:
           - 2.6
           - 2.7
+          - 3.0
```

```suggestion
        ruby: ["2.6", "2.7", "3.0"]
```

(this is mostly about a change from `3.0` to `"3.0"` to make sure it doesn't interpret it as just `3` and use the latest 3.x version)

---

Numbers are indeed loaded as floats, so better safe than sorry I guess :grin: